Gemfile

```diff
@@ -9,17 +7,30 @@ end
 
 gem 'puma', '~> 3.7'
 gem 'rails', '~> 5.1.3'
+gem 'sidekiq'
 gem 'sqlite3'
 
 group :development, :test do
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'factory_bot_rails'
+  gem 'faker'
+  gem 'rspec-rails'
 end
 
 group :development do
+  gem 'brakeman'
```

Comment:
To check for any security hoes i have in the system this is a must with all my projects